job-service-db/src/main/resources/procedures/dropTaskTables.sql

```diff
@@ -47,8 +48,9 @@ BEGIN
             PERFORM internal_drop_task_tables(in_partition_id, in_task_id || subtask_suffix);
         END LOOP;
 
-        -- Drop the table itself
-        EXECUTE 'DROP TABLE ' || task_table_ident;
+        -- Insert table name to be dropped later
+        task_table_name := internal_get_task_table_name(in_partition_id, in_task_id);
```

Comment:
We shouldn't be calling `internal_get_task_table_name(in_partition_id, in_task_id)` twice in the same function.  You should just set the `task_table_name` and then use it when setting `task_table_ident`.

Reply:
Addressed